pkg/sdk/testint/external_volumes_gen_integration_test.go

```diff
@@ -0,0 +1,605 @@
+package testint
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/testenvs"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInt_ExternalVolumes(t *testing.T) {
+	client := testClient(t)
+	ctx := testContext(t)
+
+	awsBaseUrl := testenvs.GetOrSkipTest(t, testenvs.AwsExternalBucketUrl)
+	awsRoleARN := testenvs.GetOrSkipTest(t, testenvs.AwsExternalRoleArn)
+	awsKmsKeyId := testenvs.GetOrSkipTest(t, testenvs.AwsExternalKeyId)
+	awsExternalId := "123456789"
+
+	gcsBaseUrl := testenvs.GetOrSkipTest(t, testenvs.GcsExternalBuckerUrl)
+	gcsKmsKeyId := "123456789"
+
+	azureBaseUrl := testenvs.GetOrSkipTest(t, testenvs.AzureExternalBucketUrl)
+	azureTenantId := testenvs.GetOrSkipTest(t, testenvs.AzureExternalTenantId)
+
+	assertExternalVolumeShowResult := func(t *testing.T, s *sdk.ExternalVolume, name sdk.AccountObjectIdentifier, allowWrites bool, comment string) {
```

Comment:
nit-pick: we try to use the newly generated object assertions (even in the integration tests). You can check e.g. `users_integration_tests`.

Let's leave it like this for now, we will generate these assertions later on while working on the resource